src/components/StreamerConnection.vue

```diff
@@ -59,6 +59,12 @@ export default {
       false,
     );
   },
+  onDestory() {
+
+    clearInterval(this.connection.looper);
+    this.connection = null;
```

Comment:
.....this line is probably what was fixing the issue (prior to the `onDestory` typo :))